ethereum/contracts/ETHAtomicSwapper.sol

```diff
@@ -86,48 +74,46 @@ contract ETHAtomicSwapper {
         bytes20 _bep2Addr,
         uint256 _bep2Amount
     ) external onlyInvalidSwaps(_secretHashLock) payable returns (bool) {
-        // Assume average block time interval is 15 second
+        // Assume average block time interval is 10 second
```

Comment:
is it 13 second per block?

Reply:
The reason why we assume 10 second a block is that we want to figure out a memorable height span limitation.